index.bs

```diff
@@ -340,12 +334,13 @@ An event-level trigger configuration is a [=struct=] with the following items:
 :: A 64-bit integer.
 : <dfn>filters</dfn>
 :: An [=attribution filter map=].
+: <dfn>negated filters</dfn>
```

Comment:
I believe this is being addressed on https://github.com/WICG/attribution-reporting-api/pull/435, but both PRs look ccompatible.